sale_financial_risk_info/models/res_partner.py

```diff
@@ -1,10 +1,36 @@
 # Copyright 2021 Tecnativa - Carlos Dauden
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
     risk_percent_warning = fields.Integer(string="Risk percent warning", default=80)
+
+    risk_remaining_value = fields.Monetary(
```

Comment:
I still think these fields shouldn't go in any `sale_financial_*` module, as they are general values

Reply:
@pedrobaeza in which module do you suggest we could create these fields?

Reply:
@pedrobaeza when you can, if you coul provide directions we'll try to close this PR :)